python/GangaLHCb/Lib/Applications/BenderBox.py

```diff
@@ -0,0 +1,520 @@
+
+    Note:
+    1. The ``scripts'' are executed within ``ostap'' context
+    2. ``--no-color'' and ``--batch'' options are added automatically
+
+    """
+    _schema = Schema(Version(1, 0), {
+        # Options created for constructing/submitting this app
+        'tool'             : ComponentItem (
+        category   = 'BenderTool' ,
+        defvalue   = BenderModule()      ,
+        doc        = """Bender ``tool'' to be used. See plugins[``BenderTool''] for allowed values"""
+        ),
+        'directory'        : SimpleItem    (
+        defvalue   = ''          ,
+        typelist   = [None, str] ,
+        comparable = 1           ,
+        doc        ="""A path/directory to the project that you\'re wanting to run.""" ) ,
+        'platform'         : SimpleItem(defvalue='x86_64-slc6-gcc49-opt', typelist=[str], doc='Platform the application was built for'),
+        ## from GaudiExec 
+        'uploadedInput'    : GangaFileItem(defvalue=None, hidden=1, doc='This stores the input for the job which has been pre-uploaded so that it gets to the WN'),
+        'jobScriptArchive' : GangaFileItem(defvalue=None, hidden=1, copyable=0, doc='This file stores the uploaded scripts which are generated fron this app to run on the WN'),
+        'useGaudiRun'      : SimpleItem(defvalue=False, hidden = 1, doc='Should be False to call getWNPythonContents function'),
+        'extraOpts'        : SimpleItem(defvalue='', typelist=[str], hidden=1, doc='DO NOT USE IT; An additional string which is to be added to \'options\' when submitting the job'),
+        'options'          : GangaFileItem ( defvalue=[], sequence=1, hidden = 1,doc='List of files which contain the options to pass to gaudirun.py'),
+        'extraArgs'        : SimpleItem(defvalue=[], typelist=[list], sequence=1, hidden=1, doc=' Do NOT USE IT; Extra runtime arguments which are passed to the code running on the WN'),
+        # Prepared job object
+        'is_prepared'      : SimpleItem(defvalue=None, strict_sequence=0, visitable=1, copyable=1, hidden=0, typelist=[None, ShareDir], protected=0, comparable=1,
+                                        doc='Location of shared resources. Presence of this attribute implies the application has been prepared.'),
+        'hash'             : SimpleItem(defvalue=None, typelist=[None, str], hidden=1, doc='MD5 hash of the string representation of applications preparable attributes'),
+        })
+    _category      = 'applications'
+    _name          = 'BenderBox'
+    _exportmethods = ['prepare', 'unprepare' ]
+
+    def configure(self, masterjobconfig):	
+        self.options = self.tool.inputfiles()
+        return (None,None)
+
+    # =======================================================================
+    def getWNPythonContents(self):
+        """Return the wrapper script which is used to run Bender on the WN
+        """
+        from ..RTHandlers.GaudiExecRTHandlers import GaudiExecDiracRTHandler
```

Comment:
Why the `..`? Should this rather be `GangaLHCb.Lib.RtHandlers.GaudiExecRTHandlers` ?